ux.symfony.com/templates/components/CodeLine.html.twig

```diff
@@ -1,8 +1,8 @@
-{% props highlight = false, code = '', height = null %}
+{% props code = '', height = null %}
```

Comment:
I broke highlighting here and was unsure on how to fix.

@smnandre, what do you think about merging this component with `CodeBlock` to keep it simple? (again, followup PR)